From a88571dae82576ee4233d2017cc5f7fbf89a1454 Mon Sep 17 00:00:00 2001
From: Feanil Patel <feanil@tcril.org>
Date: Thu, 23 Feb 2023 13:59:15 -0500
Subject: [PATCH 0995/1021] build: Creating a missing workflow file
 `add-remove-label-on-comment.yml`.

The .github/workflows/add-remove-label-on-comment.yml workflow is missing or needs an update to stay in
sync with the current standard for this workflow as defined in the
`.github` repo of the `openedx` GitHub org.
---
 .../workflows/add-remove-label-on-comment.yml | 20 +++++++++++++++++++
 1 file changed, 20 insertions(+)
 create mode 100644 .github/workflows/add-remove-label-on-comment.yml

diff --git a/.github/workflows/add-remove-label-on-comment.yml b/.github/workflows/add-remove-label-on-comment.yml
new file mode 100644
index 0000000..0f369db
--- /dev/null
+++ b/.github/workflows/add-remove-label-on-comment.yml
@@ -0,0 +1,20 @@
+# This workflow runs when a comment is made on the ticket
+# If the comment starts with "label: " it tries to apply
+# the label indicated in rest of comment.
+# If the comment starts with "remove label: ", it tries
+# to remove the indicated label.
+# Note: Labels are allowed to have spaces and this script does
+# not parse spaces (as often a space is legitimate), so the command
+# "label: really long lots of words label" will apply the
+# label "really long lots of words label"
+
+name: Allows for the adding and removing of labels via comment
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  add_remove_labels:
+    uses: openedx/.github/.github/workflows/add-remove-label-on-comment.yml@master
+
-- 
2.34.1

